paasta_tools/cli/cmds/security_check.py

```diff
@@ -47,6 +51,10 @@ def perform_security_check(args):
         return 0
 
     ret_code, output = _run(security_check_command, timeout=300, stream=True)
-    paasta_print("{} exited with code {}".format(security_check_command, ret_code))
+    paasta_print("{} exited with code {}:{}".format(security_check_command, ret_code, output))
+
+    sensu_status = pysensu_yelp.Status.CRITICAL if ret_code else pysensu_yelp.Status.OK
```

Comment:
I think if ret_code and if ret_code != 0 are equivalent (except for when ret_code is None), but I could use the latter for more clarity.